server/src/migration/1554142210463-AskedQuestion.ts

```diff
@@ -1,35 +0,0 @@
-import { MigrationInterface, QueryRunner, Table } from 'typeorm';
-
-export class AskedQuestion1554142210463 implements MigrationInterface {
```

Comment:
Why you delete this migration? This could break the migration flow.
If you want to create a new table with the same name, you should delete this table in next migration and create a new one.

Reply:
The typing was invalid, so it probably just was not completed for the previous migration. I am not very familiar with the subject, but If you are suggesting that just changing previous file would be enough, I will do so

Reply:
Agree, If this migration not completed at all then you could delete it.

Reply:
So I've contacted Viktor Molokostov about this and he has confirmed that the table has not been generated